football/src/main/scala/com/gu/mobile/notifications/football/Lambda.scala

```diff
@@ -62,15 +63,16 @@ object Lambda extends Logging {
 
   lazy val articleSearcher = new ArticleSearcher(capiClient)
 
-  def debugSetTime(): Unit = {
-    // this is only used to debug
-    if (configuration.stage == "CODE") {
+  def getZonedDateTime(): ZonedDateTime = {
+    val zonedDateTime = if (configuration.stage == "CODE") {
       val is = new URL("https://hdjq4n85yi.execute-api.eu-west-1.amazonaws.com/Prod/getTime").openStream()
       val json = Json.parse(Source.fromInputStream(is).mkString)
-      val date = DateTime.parse((json \ "currentDate").as[String])
-      logger.info(s"Force the date to $date")
-      DateTimeUtils.setCurrentMillisFixed(date.getMillis)
+      ZonedDateTime.parse((json \ "currentDate").as[String])
+    } else {
+      ZonedDateTime.now()
     }
+    logger.info(s"Using date time: $zonedDateTime")
+    zonedDateTime
```

Comment:
Previously this would only log in CODE, now this logs in both CODE and PROD - is that intended?

Reply:
Yep, there was a bit of refactoring here around the date/time libraries in this PR, so I want to make sure everything is working as expected when this is released to production.